tools/rosbash/rosfish

```diff
@@ -354,5 +788,14 @@ complete -x -c rosrun -a '(_roscomplete_rosrun)'
 complete -x -c roslaunch -a '(_roscomplete_launch)'
 complete -x -c rosed -a '(_roscomplete_file)'
 complete -c rosed -l help --description "prints command info"
-complete -x -c roscp -a '(_roscomplete_file)'
+#complete -x -c roscp -a '(_roscomplete_file)'et -x TEST works
```

Comment:
This change looks weird. Please check why it is commented out and what the appended string is.

Reply:
Sorry, that's because I started working on this before #171.  I commented it out (because that line isn't valid fish, as fixed in #171), the appended string is gibberish which was in there before #171 for some reason.  I'll fix it.

Reply:
The reason I left it commented out is because the implementation of `roscp` here doesn't work (and doesn't look like it should work in fish to me), and I haven't fixed it.  I'll remove the gibberish though.  Has this script (before this pr) been tested by anyone with a recent version of fish?